en_US/data-integration/rules.md

```diff
@@ -1,27 +1,49 @@
 # Rule Engine
 
-The Rules Engine is EMQX's built-in SQL-based data processing component, which works with [data-bridges](./data-bridges.md) can be used to help extract, filter, enrich, transform and store IoT data in real-time to accelerate application integration and business innovation.
+The Rule Engine is EMQX's built-in data processing component, which works with [data bridges](./data-bridges.md) and can be used to help extract, filter, enrich, transform and store IoT data to accelerate application integration and business innovation. 
 
-![image](./assets/rules/data-integration-arch.png)
+<img src="./assets/rule-engine.png" alt="SQL-based IoT Rule Engine" style="zoom:30%;" />
 
-## Composition of Rules
+The Rule Engine is particularly useful for organizations that need to process and analyze data from multiple sources, such as IoT devices or other sensor networks. By using the Rule Engine, organizations can create rules that filter out irrelevant data, perform calculations, and trigger alerts or notifications based on specific events or conditions.
```

Comment:
> organizations

I think it s better if we can avoid using the word organization. EMQX might be used by individuals as well. I think one can rephrase so one does not need to mention the type of entity that is using EMQX.

> The Rule Engine is particularly useful for organizations that need to process and analyze data from multiple sources, such as IoT devices or other sensor networks.

This sentence seems a bit meaningless to me. Practically all use cases of EMQX have to do with processing and analyzing data from multiple sources. Instead I would write something like this:

The rule engine is used to  transform and reroute incoming messages. For example, the rule engine can be used to reroute messages to a certain topic to another topic or an external service. The rule engine can also be used to filter messages and do data transformations (such as adding or removing fields).